### PR TITLE
Career Ladders

### PR DESCRIPTION
**Pull request by caitlinmoorman**
First draft of Career Ladders post. It's a bit long - would welcome any feedback on fat you'd recommend trimming (and anything else!).

### CODE REVIEW

File: content/post/career_ladders.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+## What Is a Career Ladder?
+A career ladder is a framework to document the path or paths an individual can take as they develop within their company. It describes different roles (hopefully both individual contributor roles and manager roles), outlines the expectations for each, and provides sufficiently detailed information to see what is needed to move from one role to another. A career ladder can include hard skills, as well as an outline of *how* one should work in each role and the type of impact that role should have.
+
+## Why Clear Career Paths Matter
+Analysts, of all people, want to know how they are being measured and how to reach the next level. Clearly outlining the path ahead is critical to keeping strong performers. Sometimes, it is explicitly about how to get the next promotion, but it’s also about how to increase impact, measure growth, and think about future options.
```
Comment (mikekaminsky) [inline content/post/career_ladders.md]: Nit: 

>is critical to keeping strong performers.

Maybe sub "keeping" for "retaining" or extend the sentence to something like

> is critical to keeping strong performers engaged and progressing

File: content/post/career_ladders.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## How to Create a Career Ladder
+I separate a good career ladder into two parts: guiding principles and specific competencies.
+
+### Guiding Principles
+Guiding principles are simple heuristics that capture how roles differ from each other. You don’t get promoted for just getting better at the same job, you get promoted for showing that you can do a different job.
```
Comment (mikekaminsky) [inline content/post/career_ladders.md]: 💯

File: content/post/career_ladders.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+When I think about how analytics team members work at different levels and how to clearly articulate the difference between roles, I focus on two areas: scope of influence (how wide the impact of your work is), and what deliverables look like (the types of work you do, and what done looks like). Here is my matrix for those principles for individual contributors:
+{{< figure src="/img/annotations-google-sheet.jpg" title="Guiding principles for individual contributors">}}
+
+Guiding principles set the stage for the specific competencies that comprise the rest of the ladder. But importantly, they also enable you to create flexibility and variations of the ladder as needed. You can modify a career ladder for various flavors - for example, I’ve had members of my team who strongly wanted to follow a specialist path. I manage fairly small teams and am happy to accommodate different career paths; knowing the guiding principles allowed me to create a Product Analyst ladder and a Sales Operations Analyst ladder that spoke to their skills more specifically, but still aligned very clearly with our overall levelling strategy.
```
Comment (mikekaminsky) [inline content/post/career_ladders.md]: > our overall levelling strategy.

I think you're referring to "the company's" overall levelling strategy, in which case you can clarify to

> the company's overall levelling strategy.

**Comment by sbreitenother** [comment]
@caitlinmoorman let me know where you land re splitting the post! Don't want to review if you're going to make major changes!

**Comment by caitlinmoorman** [comment]
@mikekaminsky @sbreitenother I'm going to experiment with splitting it up into multiple posts. I tried 2 posts while I was writing it and didn't like how I split it, but will give it another whirl.

**Pull request closed without merging**